packages/block-library/src/paragraph/style.scss

```diff
@@ -28,6 +28,11 @@
 	font-style: normal;
 }
 
+// Prevent the dropcap from breaking out of the box when a background is applied.
+p.has-drop-cap.has-background {
```

Comment:
Shouldn't this happen regardless of background?

Reply:
We've been a bit back and forth on this one. If we do this, then it ends up looking like this:

<img src="https://user-images.githubusercontent.com/1204802/106725302-83fbfb00-6609-11eb-92a4-fd8972ce6ed2.png">

But I would argue that the 2nd paragraph should be allowed to wrap around the dropcap, like this:

<img src="https://user-images.githubusercontent.com/1204802/106725334-8b230900-6609-11eb-89bb-367245d87425.png">

By limiting it to a paragraph that has a background, at least we allow this:

<img width="1103" alt="Screenshot 2021-03-01 at 11 49 21" src="https://user-images.githubusercontent.com/1204802/109487221-3649a580-7a84-11eb-80ee-5a5263acf380.png">